src/shared/components/MarkDownEditor/MarkDownEditor.utils.ts

```diff
@@ -78,177 +66,53 @@ MdEditor.config({
       },
     },
   },
-  markedRenderer(renderer) {
-    renderer.image = (href, title, text) => {
-      const videoExtensionMatch = href?.match(getValidMediaFormatsRegex(VALID_VIDEO_FILE_TYPES));
-      const videoLinkMatch = href?.match(VIDEO_LINK_REGEX);
-      const audioExtensionMatch = href?.match(getValidMediaFormatsRegex(VALID_AUDIO_FILE_TYPES));
-      if (videoExtensionMatch) {
-        return `${
-          text ? `<figure><figcaption>${text}:</figcaption>` : ''
-        }<video controls width="250"><source src="${href}"></video></figure>`;
-      }
-      if (audioExtensionMatch) {
-        return `${
-          text ? `<figure><figcaption>${text}:</figcaption>` : ''
-        }<audio controls><source src="${href}"></audio></figure>`;
-      }
-      if (videoLinkMatch) {
-        const defaultReturn = href ? `<a href="${href}" target="_blank">${text || href}</a>` : '';
-        const videoSource = videoLinkMatch[1];
-        const videoUrl = videoLinkMatch[3];
-        if (!videoSource || !videoUrl) return defaultReturn;
-
-        if (
-          videoSource.includes(VideoSources.Youtube) ||
-          videoSource.includes(VideoSources.Youtu)
-        ) {
-          const videoId = videoUrl.match(YOUTUBE_VIDEO_ID_REGEX)?.[1];
-
-          return videoId ? getVideoIframe(videoId, VideoSources.Youtube, text) : defaultReturn;
-        } else if (videoSource.includes(VideoSources.Vimeo)) {
-          const videoId = videoUrl.replace(/\D/g, '');
-
-          return videoId ? getVideoIframe(videoId, VideoSources.Vimeo, text) : defaultReturn;
-        }
+  markdownItConfig(mdit) {
+    mdit
+      .set({
+        html: true, // Enable HTML tags in source
+        xhtmlOut: true, // Use '/' to close single tags (<br />).
+        breaks: true, // Convert '\n' in paragraphs into <br>
+        langPrefix: 'lang-', // CSS language prefix for fenced blocks.
+        linkify: false,
+        typographer: true,
+        quotes: '“”‘’',
+      })
+      .use(emoji)
+      .use(taskLists)
+      .use(sup)
+      .use(sub)
+      .use(container)
+      .use(container, 'hljs-left') /* align left */
+      .use(container, 'hljs-center') /* align center */
+      .use(container, 'hljs-right') /* align right */
+      .use(deflist)
+      .use(abbr)
+      .use(footnote)
+      .use(insert)
+      .use(mark)
+      .use(container)
+      .use(miip)
+      .use(katex)
+      .use(html5Embed, {
+        html5embed: {
+          useImageSyntax: true,
+        },
+      })
+      .use(markdownItImSize);
+  },
+  markdownItPlugins(plugins) {
+    return plugins.map((p) => {
+      if (p.type === 'image') {
+        return {
+          ...p,
+          options: {
+            ...p.options,
+            classes: 'my-class',
```

Comment:
maybe use more special class name (as I understood here is adding class to the image)?